### PR TITLE
Replace Console with Logger almost everywhere

### PR DESCRIPTION
**Pull request by Maddin-M**
First time opening a PR on another persons repo on Github, hope I did it right. Also my (pretty strict) Lint plugin really wants to go crazy on this project 😄

### CODE REVIEW

File: korio/src/commonMain/kotlin/com/soywiz/korio/time/TraceTime.kt [hunk 1]
```diff
@@ -13,7 +13,7 @@ inline fun <T : Any> traceTime(name: String, block: () -> T): T {
         result = block()
     }
     if (traceTimes) {
-        Console.info("$name loaded in $time")
+        Logger("TraceTime").info { "$name loaded in $time" }
```
Comment (soywiz) [inline korio/src/commonMain/kotlin/com/soywiz/korio/time/TraceTime.kt:16]: Maybe @PublishedAPI internal val traceTimeLogger outside the function?

File: korio/src/jsMain/kotlin/com/soywiz/korio/runtime/node/JsRuntimeNode.kt [hunk 2]
```diff
@@ -163,6 +165,7 @@ private class NodeJsAsyncClient(val coroutineContext: CoroutineContext) : AsyncC
 private class NodeJsAsyncServer : AsyncServer {
     private val net = require_node("net")
     private var server: dynamic = null
+    private val logger = Logger("NodeJsAsyncServer")
```
Comment (soywiz) [inline korio/src/jsMain/kotlin/com/soywiz/korio/runtime/node/JsRuntimeNode.kt:168]: Maybe inside a companion object, so the calling happens just once?

**Comment by soywiz** [comment]
Nice! Just a couple minor issues.

Glad you were able to set it up and overcome the linting that is provably OCR-ish and make the PR

File: korio/src/jsMain/kotlin/com/soywiz/korio/runtime/node/JsRuntimeNode.kt (hunk 2 above)
Comment (Maddin-M) [reply korio/src/jsMain/kotlin/com/soywiz/korio/runtime/node/JsRuntimeNode.kt]: done (also for asyncClient)

File: korio/src/commonMain/kotlin/com/soywiz/korio/time/TraceTime.kt (hunk 1 above)
Comment (Maddin-M) [reply korio/src/commonMain/kotlin/com/soywiz/korio/time/TraceTime.kt]: done

**Comment by soywiz** [comment]
LGTM. Thanks! ☺️

**Pull request merged**